File: API/Models/EscapeRoom/EscapeRoom.cs
using System;

namespace API.Models.EscapeRoom
{
    public class EscapeRoom
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Street { get; set; }
        public string City { get; set; }
        public string State { get; set; }
        public string ZipCode { get; set; }
    }

    public class VisitedEscapeRoom
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public DateTime? DateVisited { get; set; }
    }
}

File: react-ui/src/Models/EscapeRoom.ts
export interface EscapeRoom {
  id: number;
  name: string;
  street: string;
  city: string;
  state: string;
  zipCode: string;
}

export interface VisitedEscapeRoom {
  id: number;
  name: string;
  dateVisited: Date | null;
}